content/_includes/CLI/InteractiveArgsEditor.md

```diff
@@ -0,0 +1,22 @@
+---
+---
+
+The TUI allows you to configure command properties and includes information on required properties, defaults, and expected input types (string, boolean, integer, or array).
+This can help when entering complex commands with multiple configurable properties.
+
+In the TUI, optional properties are disabled (with the `#` symbol) by default.
```

Comment:
is is correct to say "indicated by the `#` symbol"? if yes I recommend this change.